### ISSUE TITLE
[feat/Q] Only runing for the latest release

### ISSUE THREAD

**Issue by liangyuanpeng** [rajatjindal/krew-release-bot#75]
thanks for using krew-release-bot

if you are having issue generating correct template for your plugin, please try using https://rajatjindal.com/tools/krew-release-bot-helper

if you are having some other trouble or the above link does not help, please feel free to open the issue

According to the following release order:

1.10.3 08/27 make it to latest release
1.10.4 08/28
1.9.6 08/29

Expected effect: update-krew-index only submits PR for `1.10.3` and `1.10.4` and ignores the old version `1.9.6`, now, it's open a PR for all

Is the concept of version control under the responsibility of krew-release-bot? Because I think this is a problem that all krew plugins will face.

**Comment by ahmetb** [comment]
Krew only allows publishing the latest version.

**Comment by liangyuanpeng** [comment]
> Krew only allows publishing the latest version.

Yes, it's the reason of this issue opened,

**Comment by ahmetb** [comment]
Yeah it's by design and not related to this tool. It's also a won't fix on Krew.
